torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -398,11 +398,12 @@ void IrPrinter::handle(const ReductionOp* rop) {
   indent() << "   = reduction( " << rop->in()
            << ", op = " << rop->getReductionOpType()
            << ", initial value = " << rop->init()
-           << ", allreduce = " << rop->isAllreduce() << " )\n";
+           << ", allreduce = " << (rop->isAllreduce() ? "true" : "false")
```

Comment:
All the changes in this file are mostly cleanup, except those newly added for `GroupedWelfordOp` and `GroupedGridWelford`. The cleanup is meant to make the printing more consistently done, whereas previously the KIR classes had been formatted rather differently.